src/clipboard_handlers/cell_clipboard.ts

```diff
@@ -32,9 +33,11 @@ export class CellClipboardHandler extends AbstractCellClipboardHandler<
     return CommandResult.Success;
   }
 
-  copy(data: ClipboardCellData): ClipboardContent | undefined {
+  copy(
+    data: ClipboardCellData,
+    mode: ClipboardCopyOptions = "copyPaste"
+  ): ClipboardContent | undefined {
```

Comment:
Alternatively, we could have created a new separate clipboard handler that handle copying part of the array formula, and not loaded this clipboard on INSERT_CELL. Would have made the code more complex tho, so not sure it's worth.

Reply:
How about a new option argument (similar to the `isCutOperation` on the paste method) that we call `skipSpilledCells` 

achieves the same but haveing a method 'copy' that specifically has a different mode than 'copy' looks weird.

It zould be nice if the commit message also mentionned the "move_header" command

Reply:
Table style are not spilled cells tho :/ I don't have better ideas for a name

Reply:
yup ...